extensions/amp-document-recommendations/0.1/amp-document-recommendations.js

```diff
@@ -164,6 +164,8 @@ export class AmpDocumentRecommendations extends AMP.BaseElement {
     return this.getVsync()
         .mutatePromise(() => {
           const shadowRoot = this.win.document.createElement('div');
+          const marker = doc.createElement(MARKER_TAG);
+          doc.body.appendChild(marker);
 
           try {
```

Comment:
What throws in this try/catch? Would it be ok if the error is emitted uncaught? It might be more helpful to developers if the error was allowed through instead of silenced.